### ISSUE TITLE
Error: Unsupported source map encoding charset=utf8;base64

### ISSUE THREAD

**Issue by sydneyitguy** [postcss/postcss#799]
`Error: Unsupported source map encoding charset=utf8;base64`
I tried changing following line on `postcss/lib/previous-map.js:71`
```
let utf64 = 'data:application/json;charset=utf-8;base64,';
=> let utf64 = 'data:application/json;charset=utf8;base64,'; // NOTE utf8 instead of utf-8
```
temporarily fixed the issue.
I guess that should have both charset names to check.

```
events.js:154
      throw er; // Unhandled 'error' event
      ^
Error: Unsupported source map encoding charset=utf8;base64
  at PreviousMap.decodeInline ([...]/node_modules/postcss/lib/previous-map.js:71:19)
  at PreviousMap.loadMap ([...]/node_modules/postcss/lib/previous-map.js:91:25)
  at new PreviousMap ([...]/node_modules/postcss/lib/previous-map.js:33:25)
  at new Input ([...]/node_modules/postcss/lib/input.js:39:19)
  at parse ([...]/node_modules/postcss/lib/parse.js:22:17)
  at new LazyResult ([...]/node_modules/postcss/lib/lazy-result.js:61:24)
  at Processor.process ([...]/node_modules/postcss/lib/processor.js:34:16)
  at DestroyableTransform._transform ([...]/node_modules/gulp-autoprefixer/index.js:20:31)
  at DestroyableTransform.Transform._read ([...]/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:159:10)
```

packages:
```
"devDependencies": {
    "del": "^2.2.0",
    "gulp": "^3.9.1",
    "gulp-autoprefixer": "^3.1.0",
    "gulp-changed": "^1.3.0",
    "gulp-coffee": "^2.3.2",
    "gulp-coffeelint": "^0.6.0",
    "gulp-concat": "^2.6.0",
    "gulp-imagemin": "^2.4.0",
    "gulp-rename": "^1.2.2",
    "gulp-sass": "^2.3.1",
    "gulp-sourcemaps": "^2.0.0-alpha",
    "gulp-uglify": "^1.5.3"
  },
```

**Comment by ai** [comment]
Done 215ed88

**Comment by ai** [comment]
BTW, how did you get this wrong `utf8` encoding name? Who generate your input source map?

**Comment by ai** [comment]
Released in 5.0.21

**Comment by josephfusco** [comment]
Just wanted to point out this also happened to myself today after installing `"gulp-sourcemaps": "^2.0.0-alpha"`

**Comment by ai** [comment]
@josephfusco so 5.0.21 will fix compatibility with `gulp-sourcemaps 2.0.0-alpha`? Awesome :)
